website/docs/r/federated_database_instance.html.markdown

```diff
@@ -12,17 +12,50 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
-## Example Usages
+## Example Usages with MongoDB Atlas Cluster as storage database
 
 
 ```terraform
 resource "mongodbatlas_federated_database_instance" "test" {
   project_id         = "PROJECT ID"
   name = "NAME OF THE FEDERATED DATABASE INSTANCE"
-  aws {
-    role_id = "AWS ROLE ID"
-    test_s3_bucket = "S3 BUCKET NAME"
+  storage_databases {
+    name = "VirtualDatabase0"
+    collections {
+      name = "NAME OF THE COLLECTION"
+      data_sources {
+          collection = "COLLECTION IN THE CLUSTER"
+          database = "DB IN THE CLUSTER"
+          store_name =  "CLUSTER NAME"
+      }
+    }
   }
+
+  storage_stores {
+    name = "STORE 1 NAME"
+    cluster_name = "CLUSTER NAME"
+    project_id = "PROJECT ID"
+    provider = "atlas"
+    read_preference {
+      mode = "secondary"
+    }
+  }
+}
+```
+
+## Example Usages with AWS S3 bucjet as storage database
```

Comment:
```suggestion
## Example Usages with Amazon S3 bucket as storage database
```

Reply:
small typo, also name of service is Amazon S3 (not AWS S3) - https://aws.amazon.com/s3/